### ISSUE TITLE
Review Public API for Notifications

### ISSUE THREAD

**Issue by xavierchanth** [atsign-foundation/at_client_sdk#531]
**Is your feature request related to a problem? Please describe.**

You can't implement the notify verb without manually importing `src/service/notification_service.dart`

**Describe the solution you'd like**

At the bare minimum export NotificationParams from at_client.

**Describe alternatives you've considered**

Maybe also export NotificationService as well so you can do something like:

```dart
AtClientManager atClientManager = AtClientManager.getInstance();
AtClient atClient = atClientManager.atClient;
NotificationService notificationService = atClientManager.notificationService;

atClient.put(...);
notificationService.notify(...);
```

**Additional context**
N/A

**Comment by xavierchanth** [comment]
@gkc @VJag I would love some feedback on this, I can go ahead and make the changes based on your input.

**Comment by xavierchanth** [comment]
Closed via #540
